implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/DynatraceMeterRegistry.java

```diff
@@ -89,6 +98,22 @@ protected TimeUnit getBaseTimeUnit() {
         return this.exporter.getBaseTimeUnit();
     }
 
+    @Override
+    protected DistributionSummary newDistributionSummary(Meter.Id id, DistributionStatisticConfig distributionStatisticConfig, double scale) {
+        if (apiVersion == DynatraceApiVersion.V2) {
+            return new DynatraceDistributionSummary(id);
+        }
+        return super.newDistributionSummary(id, distributionStatisticConfig, scale);
+    }
+
+    @Override
+    protected Timer newTimer(Meter.Id id, DistributionStatisticConfig distributionStatisticConfig, PauseDetector pauseDetector) {
+        if (apiVersion == DynatraceApiVersion.V2) {
+            return new DynatraceTimer(id, clock, exporter.getBaseTimeUnit());
```

Comment:
Shouldn't this be aware both the distributionStatisticConfig and the pauseDetector?